x-pack/plugins/lens/public/datatable_visualization/components/table_basic.scss

```diff
@@ -1,3 +1,8 @@
+.lnsDataTableContainer {
+  height: 100%;
+  overflow: initial;
```

Comment:
Is this `overflow` style necessary? It appears to be getting overridden by styles from `.lnsVisualizationContainer`. I assume it can be removed.